### DISCUSSION TITLE
The mixed chart histogram indicates the order adjustment of the items

### DISCUSSION THREAD

**Discussion by cuipeng1025** [apache/superset#27438] [General]
### Bug description

I have two items, the expected sales amount and the actual sales amount. I want to use a bar chart for them, and specify the expected amount to be measured on the left and the actual amount to be measured on the right. How should I specify the expected amount to be measured?
<img width="474" alt="スクリーンショット 2024-03-08 093108" src="https://github.com/apache/superset/assets/160709735/078df036-3a20-481e-a8eb-0c825322a79b">

### How to reproduce the bug

Generally, this bug can be reproduced by making a mixed chart histogram. If it is not implemented once, it can be reproduced by modifying the presentation order of the data items.

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by michael-s-molina** [comment]
Converting to a GitHub discussion given:

> 🙏 Asking a question or getting help: post in the [Superset Slack chat](http://bit.ly/join-superset-slack) or [GitHub Discussions](https://github.com/apache/superset/discussions) under "Q&A / Help".
